core/package.json

```diff
@@ -19,16 +19,13 @@
     "@luigi-project/core": "^1.7.1",
     "@luigi-project/plugin-auth-oidc": "1.8.1",
     "copy-webpack-plugin": "^5.0.0",
-    "file-saver": "2.0.2",
-    "prettier": "^2.2.1"
+    "file-saver": "2.0.2"
   },
   "devDependencies": {
     "babel-loader": "^8.0.4",
     "sirv-cli": "^1.0.11",
-    "webpack": "^4.44.2"
-  },
-  "prettier": {
-    "singleQuote": true
+    "webpack": "^4.44.2",
+    "prettier": "^1.18.2"
```

Comment:
You can see it [here](https://status.build.kyma-project.io/log?container=test&id=1397136011120611328&job=pre-busola-core) - I've removed the prettier from `core/package.json`, and added a command to show prettier's version.

The versions is 1.15.3 (instead of at least 1.18.2, as specified in root package.json), and it errors out for some files.